float32 x                # x position [m]
float32 y                # y position [m]
string label             # obj label (optional)
std_msgs/ColorRGBA color # obj color (optional)